src/components/Layout/Layout.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { useGeneralContext } from '../../state/GeneralContext';
+import Header from '../Header';
+import Sidebar from '../Sidebar';
+import { LayoutDiv, MainContainer } from './Layout.styles';
+
+function Layout({ children }) {
+  const { displaySidebar } = useGeneralContext();
+
+  return (
+    <LayoutDiv
+      id="layout"
+      className={displaySidebar ? `sidebar-display` : `sidebar-hidden`}
+    >
+      <Header/>
+
+      {displaySidebar ? (
+        <Sidebar/>
+      ) : null}
```

Comment:
Sugiero el operador `&&` de nuevo.
```suggestion
      {displaySidebar && (
        <Sidebar/>
      )}
```